### PR TITLE
feat: Add dkp-insights-management v0.1.2

### PR DESCRIPTION
**Pull request by kaiwalyajoshi**

### CODE REVIEW

**Comment by takirala** [comment]
Testing this in kommander CI via https://github.com/mesosphere/kommander/pull/1585

**Review by branden — CHANGES_REQUESTED** [review]
LGTM, this just needs version references bumped to reflect the new version.

**Comment by takirala** [comment]
whenever this is mutated, @kaiwalyajoshi you can run @mesosphere-mergebot test all on the attached kommander PR to verify with the latest main. 

cc. @gracedo @kaiwalyajoshi @karishmagadiya do we want to create a ticket to add some tests into kommander as insights would be a default app now ?

**Review by branden — APPROVED** [review]

**Comment by kaiwalyajoshi** [comment]
>cc. @gracedo @kaiwalyajoshi @karishmagadiya do we want to create a ticket to add some kuttl tests into kommander as insights would be a default app now ?

Yes, we have KUTTL tests that create a crashlooping pod, on an attached cluster. This should result in a CRD being created on the management cluster.

We will need the full management & attached cluster pair to do a simple verification that the e2e processing works.
Insights itself can run a gamut of e2e tests before pushing new versions to Kommander.

**Review by gracedo — COMMENTED** [review]
actually - I think it would be good to eventually get all the apps to use this kind of construct, it would make adding extra jobs and kustomization dependencies easier in the future (which we had to make use of to orchestrate upgrades that needed upgrade "hooks" which we implemented using kustomizations and dependencies). Would you be able to restructure this a bit to create a kustomization that points to the helmrelease? Essentially following the model in istio app? https://github.com/mesosphere/kommander-applications/tree/main/services/istio/1.11.6

**Comment by kaiwalyajoshi** [comment]
sure, let me get that teed up.

**Comment by kaiwalyajoshi** [comment]
@gracedo the latest patch reworks this PR to the istio way.

**Comment by kaiwalyajoshi** [comment]
Kicked off a `kommander` build here: https://github.com/mesosphere/kommander/pull/1590

This has @gracedo latest cli commits: https://github.com/mesosphere/kommander-cli/pull/444 and this PR.

**Review by gracedo — APPROVED** [review]

File: services/dkp-insights-management/0.1.2/helmrelease.yaml [hunk 1]
```diff
@@ -0,0 +1,21 @@
+apiVersion: kustomize.toolkit.fluxcd.io/v1beta2
+kind: Kustomization
+metadata:
+  name: dkp-insights-management-helmrelease
+  namespace: ${releaseNamespace}
+spec:
+  force: false
+  prune: true
+  wait: true
+  interval: 6h
```
Comment (gracedo) [inline services/dkp-insights-management/0.1.2/helmrelease.yaml:10]:
```suggestion
  interval: 6h
  retryInterval: 1m
```
actually we need this one

**Review by takirala — APPROVED** [review]

**Pull request merged**